src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -1163,15 +1163,23 @@ ss::future<cluster::init_tm_tx_reply> tx_gateway_frontend::limit_init_tm_tx(
               "tx cache is at capacity; expiring oldest tx with id:{}",
               tx.id);
             auto tx_units = co_await stm->lock_tx(tx.id, "init_tm_tx");
-            auto ec = co_await do_expire_old_tx(
-              stm,
-              term,
-              tx.id,
-              config::shard_local_cfg().create_topic_timeout_ms(),
-              true);
+
+            auto timeout = config::shard_local_cfg().create_topic_timeout_ms();
```

Comment:
why are we using `create_topic_timeout` here ?

Reply:
it's a refactoring so it isn't something new but the historical reason is we need a timeout which covers happy replication: write to the disk and a cluster wide rtt; ideally we should have a new property but there are associated overhead with introducing a property and a minor release doesn't look like a place for it